src/modules/vtol_att_control/vtol_type.h

```diff
@@ -225,6 +226,7 @@ class VtolType
 	struct vehicle_local_position_s			*_local_pos;
 	struct vehicle_local_position_setpoint_s	*_local_pos_sp;
 	struct airspeed_validated_s 				*_airspeed_validated;					// airspeed
+	float *_airspeed_smooth;
```

Comment:
Is there a reason this float is a pointer? I don't see why

Reply:
I don't like it either, but chosen to follow the coding pattern of the module. See variables above.

Reply:
I think the ones above are pointers because they are structs, and I even think they have to be pointers because of the uORB API. For a counterxample, see all the float vairables below (_ra_hrate, _ra_hrate_sp, etc)